### ISSUE TITLE
Update Committers

### ISSUE THREAD

**Issue by amakarudze** [DjangoGirls/wiki#53]
Many contributers have stepped down. Need to see which committers are still active and update the details on the [who-is-who page](https://github.com/DjangoGirls/wiki/blob/main/general/people.md).

**Comment by das-g** [comment]
> Many contributers have stepped down. Need to see which committers are still active

(assuming this refers only to people with push permission (who can also merge PRs), as the title mentions "committers")

As far as I can tell, for [DjangoGirls/tutorial](https://github.com/DjangoGirls/tutorial) and [DjangoGirls/tutorial-extensions](https://github.com/DjangoGirls/tutorial) that's mostly @magul and me (@das-g). On [DjangoGirls/djangogirls](https://github.com/DjangoGirls/djangogirls) (the code backing the website https://djangogirls.org including the system where workshop organizers and potential participants can sign up and organizers publish an event website) more/other committers seem to be active.

**Comment by amakarudze** [comment]
Yes, it refers to people with push permission and can also merge PRs. 

Thanks for the info @das-g.
